torch/_dynamo/output_graph.py

```diff
@@ -920,11 +921,18 @@ def append_prefix_insts():
 
     def cleanup_graph(self):
         """
+        Remove "creation_timestamp" from node meta
+
         Remove this pattern from the graph:
             torch._C._set_grad_enabled(False)
             torch._C._set_grad_enabled(True)
         """
+        assert self.should_exit
         nodes = list(self.graph.nodes)
+        for node in nodes:
+            if "creation_timestamp" in node.meta:
+                del node.meta["creation_timestamp"]
```

Comment:
```suggestion
            node.meta.pop("creation_timestamp", None)
```
More efficient and succinct